### ISSUE TITLE
Fix your coremod downloader.

### ISSUE THREAD

**Issue by MagmaSlime123** [RedGear/RedGearCore#7]
Fix your Coremod downloader. It fails to download the mod this uses to work.

**Comment by Jonius7** [comment]
Yeah I'm not sure, their RedGearCore dev jar seems to have disappeared, from when I updated [Brewcraft](https://github.com/Jonius7/Brewcraft) in 2019. 
`http://dl.bintray.com/lordblackhole/RedGear`
I tried setting up the dev enviroment yesterday from what I had back then and there's a lot of stuff not working.

**Comment by MagmaSlime123** [comment]
May wanna post a fixed version then. Doubt the dev will give an F about this.
The link just says Forbidden, so its useless too.

**Comment by Jonius7** [comment]
I got it to work, using a private Maven repo fixing [build.gradle and gradle.properties](https://github.com/Jonius7/Brewcraft/commit/fe6be17d386611179d3dd362fa63205e1aab62f5)
Got the dev jar using the Wayback Machine: https://web.archive.org/web/20210501113153/https://dl.bintray.com/lordblackhole/RedGear/redgear/core/RedGearCore/1.7.10-2.2.2/RedGearCore-1.7.10-2.2.2-dev.jar

---
However setting up publicly (via Github) is a Continuous Integration mess that I am struggling to figure out (just some of it here): https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-gradle-registry

I could probably make it work, but it won't be the "proper" way. "real" programmers would probably be roasting me for not figuring it out.

**Comment by Jonius7** [comment]
Each mod has the same broken link and 1 file, [src/main/resources/dependencies.info](https://github.com/Jonius7/Brewcraft/blob/1.3.5.1/src/main/resources/dependencies.info), and will need to be updated separately.
Alternatively, it is very simple to do it manually (by opening up the jar and dropping in the new `dependencies.info` file).

I have now created my own public Maven repo at JFrog to host the RedGearCore for the foreseeable future.

**Comment by MagmaBro123** [comment]
Alright. But how will using RedGearCore and its submods work then if the original ones won't work.

**Comment by MagmaSlime123** [comment]
@Jonius7 So pretty much all the links and files you gave don't work.
Replacing the file you gave me with the one in Snowfall just causes a crash, and RGC also has no file.

Would be nice if you could give me the RGC file with the fixed links and Snowfall with the fixed links directly.

**Comment by Jonius7** [comment]
Oh, because the `dependencies.info` I linked you is the source file that gets the version at build time (so it has those `${xxxxx}` variables.

I have linked the zip file which contains the updated Snowfall jar, and `dependencies.info` separately:
[Snowfall-1.7.10-2.2.2.1.jar.zip](https://github.com/RedGear/RedGearCore/files/7272726/Snowfall-1.7.10-2.2.2.1.jar.zip)

As I mentioned, every other RedGear mod needs the new `dependencies.info` as well or have RedGearCore downloaded into the `mods\1.7.10` folder already

**Comment by MagmaSlime123** [comment]
Oh.

I'm not a modder and am not a big fan of messing with the internals of mods, mainly why I wanted the file directly. Sorry if I'm being bossy.

Thanks a lot, though.

**Comment by MagmaBro123** [comment]
@Jonius7 You ok if #8.

**Comment by MagmaBro123** [comment]
@Jonius7 Your link is broken. Please fix it.

**Comment by MagmaBro123** [comment]
@Jonius7 Your link is broken. Stuck with this exact issue, yet again. Please fix it.

**Comment by MagmaBro123** [comment]
@Jonius7 Your link is broken. Stuck with this exact issue, yet again. Please fix it.

**Comment by MagmaSlime123** [comment]
> Oh, because the `dependencies.info` I linked you is the source file that gets the version at build time (so it has those `${xxxxx}` variables.
> 
> I have linked the zip file which contains the updated Snowfall jar, and `dependencies.info` separately: [Snowfall-1.7.10-2.2.2.1.jar.zip](https://github.com/RedGear/RedGearCore/files/7272726/Snowfall-1.7.10-2.2.2.1.jar.zip)
> 
> As I mentioned, every other RedGear mod needs the new `dependencies.info` as well or have RedGearCore downloaded into the `mods\1.7.10` folder already


@Jonius7 Your link is broken. Stuck with this exact issue, yet again. Please fix it.

**Comment by Jonius7** [comment]
I'm not going to continue fixing it as the repository the file is on deactivates due to lack of activity
Simply include RedGearCore in your modpack
